src/wp-includes/block-bindings.php

```diff
@@ -12,10 +12,56 @@
 /**
  * Registers a new block bindings source.
  *
- * Sources are used to override block's original attributes with a value
- * coming from the source. Once a source is registered, it can be used by a
- * block by setting its `metadata.bindings` attribute to a value that refers
- * to the source.
+ * Registering a source consists of defining a **name** for that source as well as a callback function
+ * which specifies how to get a value from that source and pass it to a block attribute.
```

Comment:
please take it like a simple suggestion:

```suggestion
* Registering a source consists of defining a **name*
* for that source and a callback function specifying
* how to get a value from that source and pass it to a block attribute.
```